cicecore/cicedynB/dynamics/ice_transport_remap.F90

```diff
@@ -339,15 +340,21 @@ subroutine horizontal_remap (dt,                ntrace,     &
          ntrace       ! number of tracers in use
 
       real (kind=dbl_kind), intent(in), dimension(nx_block,ny_block,max_blocks) :: &
-         uvel       ,&! x-component of velocity (m/s)
-         vvel         ! y-component of velocity (m/s)
+         uvel       ,&! x-component of velocity (m/s) ugrid
+         vvel         ! y-component of velocity (m/s) ugrid
+
+      real (kind=dbl_kind), intent(in), optional, dimension(nx_block,ny_block,max_blocks) :: &
+         uvelE       ,&! x-component of velocity (m/s) egrid
+         vvelN         ! y-component of velocity (m/s) ngrid
```

Comment:
If uvelE and vvelN are optional, then there needs to be an 'if present' check

Reply:
Done.